[2024-09-09T16:53:14.697+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-09T16:53:14.730+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_example_dag.process_data manual__2024-09-09T16:42:12.992273+00:00 [queued]>
[2024-09-09T16:53:14.747+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_example_dag.process_data manual__2024-09-09T16:42:12.992273+00:00 [queued]>
[2024-09-09T16:53:14.748+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 5
[2024-09-09T16:53:14.772+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): process_data> on 2024-09-09 16:42:12.992273+00:00
[2024-09-09T16:53:14.784+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=86) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-09T16:53:14.787+0000] {standard_task_runner.py:72} INFO - Started process 87 to run task
[2024-09-09T16:53:14.788+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'simple_example_dag', 'process_data', 'manual__2024-09-09T16:42:12.992273+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpl08flf__']
[2024-09-09T16:53:14.790+0000] {standard_task_runner.py:105} INFO - Job 63: Subtask process_data
[2024-09-09T16:53:14.869+0000] {task_command.py:467} INFO - Running <TaskInstance: simple_example_dag.process_data manual__2024-09-09T16:42:12.992273+00:00 [running]> on host 0a0201be4888
[2024-09-09T16:53:15.015+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Denis' AIRFLOW_CTX_DAG_ID='simple_example_dag' AIRFLOW_CTX_TASK_ID='process_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-09T16:42:12.992273+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-09T16:42:12.992273+00:00'
[2024-09-09T16:53:15.017+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-09T16:53:15.058+0000] {logging_mixin.py:190} INFO - Processing Denis and role - Data Engineer
[2024-09-09T16:53:15.059+0000] {python.py:240} INFO - Done. Returned value was: Processing Denis and role - Data Engineer
[2024-09-09T16:53:15.100+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-09T16:53:15.101+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=simple_example_dag, task_id=process_data, run_id=manual__2024-09-09T16:42:12.992273+00:00, execution_date=20240909T164212, start_date=20240909T165314, end_date=20240909T165315
[2024-09-09T16:53:15.165+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-09T16:53:15.216+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-09T16:53:15.221+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-09-10T17:17:29.807+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T17:17:29.914+0000] {taskinstance.py:2602} INFO - Dependencies not met for <TaskInstance: simple_example_dag.process_data manual__2024-09-09T16:42:12.992273+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-09-10T17:17:29.921+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
